trinity/protocol/eth/peer.py

```diff
@@ -20,21 +14,28 @@
     _DecodedMsgType,
 )
 
-from trinity.protocol.base_request import BaseRequest
 from .commands import (
-    BlockHeaders,
     NewBlock,
     Status,
 )
 from . import constants
-from .requests import HeaderRequest
 from .proto import ETHProtocol
+from .handlers import ETHRequestResponseHandler
 
 
 class ETHPeer(BasePeer):
     _supported_sub_protocols = [ETHProtocol]
     sub_proto: ETHProtocol = None
 
+    _handler: ETHRequestResponseHandler = None
+
+    @property
+    def handler(self) -> ETHRequestResponseHandler:
```

Comment:
Handler alone is such a generic word: it could mean connection handler, service handler, etc.

Could it just be named for the sub protocol it uses, like:

```py
peer = EthPeer(...)
peer.eth.get_block_headers(...)
```

and

```py
peer = LESPeer(...)
peer.les.get_proof(...)
```
etc. Reading it would always be explicit about which subprotocol is being used.